source/intro.tex

```diff
@@ -16,8 +16,8 @@
 \Cpp{} provides many facilities
 beyond those provided by C, including additional data types,
 classes, templates, exceptions, namespaces, operator
-overloading, function name overloading, references, free store
-management operators, and additional library facilities.%
+overloading, function name overloading, references, allocation and
```

Comment:
This doesn't preserve the meaning of the original wording, which describes features that C++ has *beyond* C. C also has allocation and deallocation functions (`malloc` and `free`).

If we said *replaceable* or *class-specific* allocation functions, that would be a beyond-C feature.

Reply:
Or just keep the word operators, since that's what it originally said, and C doesn't have those. The operators construct and destroy objects instead of just allocating. C doesn't have that (just `calloc` which doesn't work for non-trivial types).

Reply:
> This doesn't preserve the meaning of the original wording, which describes features that C++ has _beyond_ C. C also has allocation and deallocation functions (`malloc` and `free`).

In the C++ standard wording, allocation and deallocation functions are `operator new`/`operator new[]` and `operator delete`/`operator delete[]` but not other functions. But such meaning seems confusing when using in more general contexts.

I'm now using "allocation and deallocation operator functions", as these functions are also operator functions ([[over.oper.general]/1](https://eel.is/c++draft/over.oper.general#1)) and operator functions are clearly beyond C.

Reply:
> Or just keep the word operators, since that's what it originally said, and C doesn't have those. The operators construct and destroy objects instead of just allocating. C doesn't have that (just `calloc` which doesn't work for non-trivial types).

Hmm, however, as you said here, the `new` and `delete` operators are more than "free store management" which seemingly just refers to allocating and deallocating. The original meaning is unclear to me.